lms/djangoapps/verified_track_content/models.py

```diff
@@ -28,14 +28,17 @@ def move_to_verified_cohort(sender, instance, **kwargs):  # pylint: disable=unus
     """
     course_key = instance.course_id
     verified_cohort_enabled = VerifiedTrackCohortedCourse.is_verified_track_cohort_enabled(course_key)
+    verified_cohort_name = VerifiedTrackCohortedCourse.verified_cohort_name_for_course(course_key)
 
     if verified_cohort_enabled and (instance.mode != instance._old_mode):  # pylint: disable=protected-access
         if not is_course_cohorted(course_key):
             log.error("Automatic verified cohorting enabled for course '%s', but course is not cohorted", course_key)
         else:
             existing_cohorts = get_course_cohorts(get_course_by_id(course_key), CourseCohort.MANUAL)
-            if any(cohort.name == VERIFIED_COHORT_NAME for cohort in existing_cohorts):
-                args = {'course_id': unicode(course_key), 'user_id': instance.user.id}
+            if any(cohort.name == verified_cohort_name for cohort in existing_cohorts):
+                args = {
+                    'course_id': unicode(course_key), 'user_id': instance.user.id, 'cohort_name': verified_cohort_name
+                }
                 # Do the update with a 3-second delay in hopes that the CourseEnrollment transaction has been
                 # completed before the celery task runs. We want a reasonably short delay in case the learner
                 # immediately goes to the courseware.
```

Comment:
In the log message below on line 52, add in the log statement the name of the cohort we are looking for.